elastic/datadog_checks/elastic/elastic.py

```diff
@@ -370,6 +366,20 @@ class ESCheck(AgentCheck):
         "elasticsearch.fs.total.disk_write_size_in_bytes": ("gauge", "fs.io_stats.total.write_kilobytes"),
     }
 
+    ADDITIONAL_METRICS_PRE_6_3 = {
+        "elasticsearch.thread_pool.bulk.active": ("gauge", "thread_pool.bulk.active"),
+        "elasticsearch.thread_pool.bulk.threads": ("gauge", "thread_pool.bulk.threads"),
+        "elasticsearch.thread_pool.bulk.queue": ("gauge", "thread_pool.bulk.queue"),
+        "elasticsearch.thread_pool.bulk.rejected": ("rate", "thread_pool.bulk.rejected"),
+    }
+
+    ADDITIONAL_METRICS_POST_6_3 = {
```

Comment:
Those metrics should be added in the `metadata.csv` file

Reply:
You may find more information about the `metadata.csv` file here: https://github.com/DataDog/integrations-core/blob/master/docs/dev/new_check_howto.md#metadatacsv